pkg/comp-functions/functions/vshnpostgres/loadBalancer.go

```diff
@@ -104,3 +110,27 @@ func updateConnectionSecretWithLoadBalancerIP(ctx context.Context, svc *runtime.
 	svc.SetConnectionDetail("LOADBALANCER_IP", []byte(ip))
 
 }
+
+// we effectively have to allow all traffic that the service can be
+// accessed via the loadbalancer.
+func addLoadbalancerNetpolicy(svc *runtime.ServiceRuntime, comp *vshnv1.VSHNPostgreSQL) error {
```

Comment:
I think you can reuse this method across all services. Just pass `InfoGetter` resource instead of comp

Reply:
True, will do that.